CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+- `http.route` attribute to gin http server metrics in `go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin`. (#7275)
```

Comment:
We append, not prepend new changelog entries. Could you move this one to the last item in this section?